functions.php

```diff
@@ -63,6 +64,75 @@ function design_system_include_block_style_variations( $dir_path ) {
  */
 function dswp_restrict_locking_unlocking_blocks( $settings ) {
     $settings['canLockBlocks'] = current_user_can( 'activate_plugins' );
-	return $settings;
+    return $settings;
 }
 add_filter( 'block_editor_settings_all', 'dswp_restrict_locking_unlocking_blocks', 10, 2 );
+
+
+/**
+ * Disables the default patterns from WordPress.
+ */
+function disable_default_block_patterns() {
+    remove_theme_support( 'core-block-patterns' );
+}
+add_action( 'init', 'disable_default_block_patterns' );
+
+
+/**
+ * Combines any child theme's color palette with the Design System Theme Color palette.
+ *
+ * @param object $theme_json The theme.json schema.
+ */
+function custom_wp_theme_json_theme( $theme_json ) {
```

Comment:
if we are using functional programming all of our functions should have a prefix to prevent name collisions